src/network/networkd-util.c

```diff
@@ -116,6 +116,47 @@ DEFINE_STRING_TABLE_LOOKUP_FROM_STRING(dhcp_deprecated_address_family, AddressFa
 DEFINE_PRIVATE_STRING_TABLE_LOOKUP_FROM_STRING(ip_masquerade_address_family, AddressFamily);
 DEFINE_STRING_TABLE_LOOKUP(dhcp_lease_server_type, sd_dhcp_lease_server_type_t);
 
+int config_parse_link_local_address(
```

Comment:
Please move this to networkd-ipv4ll.c.
Also, maybe it is better to name this as `config_parse_ipv4ll_start_address()`? `config_parse_link_local_address()` sounds slightly ambiguous for me.

Reply:
Sure, that sounds ok for me.